shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -247,7 +247,6 @@ OffScreenRenderWidgetHostView::~OffScreenRenderWidgetHostView() {
         content::DelegatedFrameHost::HiddenCause::kOther);
   delegated_frame_host_->DetachFromCompositor();
 
-  delegated_frame_host_.reset();
   compositor_.reset();
   root_layer_.reset();
```

Comment:
Do these two calls need to be explicit in the destructor ?

Reply:
@deepak1556 good catch, I initially removed them because you're right, they're entirely redundant.

I left them alone in this PR because I have another cleanup PR in mind and figured removing them was a little off-topic for this bugfix PR.